src/components/results/securityanalysis/security-analysis.type.ts

```diff
@@ -47,7 +47,7 @@ export interface SecurityAnalysisNmkTableRow {
     contingencyEquipmentsIds?: (string | undefined)[];
     contingencyId?: string;
     limit?: number;
-    limitName?: string;
+    limitName?: string | null;
```

Comment:
why is undefined not needed here, but not for loadflow results ?

Reply:
The question mark already covers the "undefined" case.